voxgraph/src/frontend/submap_collection/voxgraph_submap.cpp

```diff
@@ -93,6 +93,10 @@ void VoxgraphSubmap::finishSubmap() {
   std::cout << "\n# isosurface vertices: " << isosurface_vertices_.size()
             << std::endl;
 
+  // Set mapping times
+  mapping_interval_.first = getStartTime().toSec();
+  mapping_interval_.second = getEndTime().toSec();
```

Comment:
Is there a reason to use seconds here? I think we should change cblox to store `uint64_t` and use nano seconds (see [here](http://docs.ros.org/latest/api/rostime/html/classros_1_1TimeBase.html#ac6c1cb1ef005601de453a14cace6be26)).

Reply:
+1
nanoseconds in `uint64_t` would be more general and cover all possible use cases :)

Reply:
Agreed, I've changed it here and in cblox in this [PR](https://github.com/ethz-asl/cblox/pull/34)